### PR TITLE
Modification in Marking duplication for DNA-mapping

### PR DESCRIPTION
**Pull request by LeilyR**
I have replaced the picard markdup with the sambamba markdup, also added a rule for sambamba flagstat and added its output to multiqc for the purpose of data visualisation. Moreover, I have also added the bowtie alignment summary in multiqc.

### CODE REVIEW

File: shared/rules/sambamba.snakefile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+           mapping_prg+"/{sample}.sorted.bam"
+       output:
+           mapping_prg+"/{sample}.bam"
+       shell:
+           sambamba_path+"sambamba_v0.6.6 "
+           "markdup "
+           "{input} "
+           "{output}"
+
+rule Flagstat:
```
Comment (vivekbhr) [inline shared/rules/sambamba.snakefile]: better rename to sambamba_flagstat

File: workflows/RNA-seq/RNA-seq [hunk 2]
```diff
@@ -29,7 +29,7 @@ def parse_args(defaults={"verbose":None,"configfile":None,"max_jobs":None,"snake
                          "mode":None, "downsample":None, "trim":None,"trim_prg":None,"trim_options":None, "fastqc":None,
                          "library_type":None, "mapping_prg":None, "star_options":None, "hisat_options":None, "salmon_index_options":None,
                          "featurecounts_options":None, "filter_annotation":None, "sample_info":None,
-                         "bw_binsize":None, "mapq":None}):
+                         "bw_binsize":None, "Genomic_contamination":None, "mapq":None}):
```
Comment (vivekbhr) [inline workflows/RNA-seq/RNA-seq]: @LeilyR are you planning to expand this in future? At this moment it's commented below

**Comment by vivekbhr** [comment]
@LeilyR shall I close this one as #86 seems to also have these changes?

**Comment by LeilyR** [comment]
sure

**Pull request closed without merging**